src/Helpers/camelCaseToScreamingSnakeCase.test.ts

```diff
@@ -0,0 +1,27 @@
+import { describe } from 'riteway'
+import { camelCaseToScreamingSnakeCase } from './camelCaseToScreamingSnakeCase'
+
+describe('camelCaseToScreamingSnakeCase()', async (should: any) => {
+  const { assert } = should()
+
+  assert({
+    given: 'no arguments',
+    should: 'return an empty string',
+    actual: camelCaseToScreamingSnakeCase(),
+    expected: ''
+  })
+
```

Comment:
Maybe add a `given empty string` test case, the other test cases look good :+1:

Reply:
I added this test.